homeassistant/components/conversation/default_agent.py

```diff
@@ -30,6 +30,7 @@
     async_should_expose,
 )
 from homeassistant.const import EVENT_STATE_CHANGED, MATCH_ALL
+from homeassistant.core import EventStateChangedData
```

Comment:
We already import just `core`. Let's fix it here instead by using `core.EventStateChangedData`.
https://github.com/home-assistant/core/blob/289700dcf5519aeab2966d46d553259d887665e8/homeassistant/components/conversation/default_agent.py#L187